test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ValidateTests.cs

```diff
@@ -55,9 +55,8 @@ public async void GivenAValidateRequestInXML_WhenTheResourceIsValid_ThenAnOkMess
         [Theory]
         [InlineData(
             "Patient/$validate",
-            "{\"resourceType\":\"Patient\",\"name\":\"test, one\"}",
-            "Type checking the data: Since type HumanName is not a primitive, it cannot have a value (at Resource.name[0])")]
-        public async void GivenAValidateRequest_WhenTheResourceIsInvalid_ThenADetailedErrorIsReturned(string path, string payload, string expectedIssue)
+            "{\"resourceType\":\"Patient\",\"name\":\"test, one\"}")]
```

Comment:
I created [#AB73194](https://microsofthealth.visualstudio.com/Health/_workitems/edit/73194) to track [#1304](https://github.com/FirelyTeam/fhir-net-api/issues/1304) in our system. Once we are unblocked, we can return the error message from the FHIR library and add this back in.